app/services/doc_auth/mock/result_response.rb

```diff
@@ -166,15 +183,23 @@ def classification_info
       end
 
       def doc_auth_result_from_success
```

Comment:
I wonder if we still need this method? It seems possible that we could refactor this away if we aren't using the `doc_auth_result` for any logic.